### ISSUE TITLE
[request] List of recently added/new packages

### ISSUE THREAD

**Issue by helmesjo** [build2/brep#2]
## TLDR
On [cppget.org](https://cppget.org/), add a list of recently added packages (last x days?).

## Reasoning
`build2` is starting to gain more and more traction, and I believe it would be relevant for people considering moving over existing projects to `build2` (me included) to just hop on [cppget.org](https://cppget.org/) and see recent development (instead of sporadically checking if any of your current 10+ dependencies have been packaged). Right now the only clue you get is whether the number of packages have increased or not.

Just my two cents!

**Comment by boris-kolpackov** [comment]
Yes, good idea. Sorting packages by most-recently-added has been on our TODO list for some time now. The main contention point is that once we do it, we cannot throw away our database state anymore -- right now it doesn't contain any precious information and can be easily reloaded.
